Source/NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -426,7 +426,11 @@ public final class NVActivityIndicatorView: UIView {
     }
 
     /// Color of activity indicator view.
-    @IBInspectable public var color: UIColor = NVActivityIndicatorView.DEFAULT_COLOR
+    @IBInspectable public var color: UIColor = NVActivityIndicatorView.DEFAULT_COLOR {
+        didSet {
+            setUpAnimation()
```

Comment:
Seems it not an appropriate way to update the color. As long as, `startAnimating()` hasn't been called, the color can be easily changed by `view.color = newColor`. Otherwise, it needs a proper way to reconstruct the animation to avoid duplicate sublayers as well as starting animating unexpectedly.